packages/scripts/src/get-changed-adapters/soakTestBlacklist.ts

```diff
@@ -35,6 +35,7 @@ export const SoakTestBlacklist: string[] = [
   'fmpcloud',
   'frxeth-exchange-rate',
   'genesis-volatility',
+  'gm-token',
```

Comment:
Can you add a comment explaining why we're blacklisting from soak tests? This should make it easier to figure out what needs to change to enable in the future

Reply:
That makes sense - leaving in the blacklist sounds fine rather than failing

```suggestion
  'gm-token', // Composite EA - Missing underlying source EAs
```